guides/v4.5.0/components/helper-functions.md

```diff
@@ -118,7 +148,7 @@ We can then use this helper in the component's template to get the first letter
 </section>
 ```
 
-### Named Arguments
+## Named Arguments
```

Comment:
To my understanding, they should work roughly similarly with local helpers:

```handlebars
{{this.calculate 1 2 op="add"}}
```

Would it be better to keep this as a separate section and insert here the difference between the using it as a local vs global helper? Or do we move all this into the separate sections created for 'Local Helper Functions' and 'Global Helper Functions'?

Reply:
I asked myself, which option is the most clear to beginners? And I think it’s to keep the explanation separate for each kind of helper. The “global helpers” section has the potential to be really short, where we can build on previous learning in this way: “Just like local helpers, you can use positional arguments.”